universe/node/api_space_attributes.go

```diff
@@ -128,14 +128,11 @@ func (n *Node) apiGetSpaceWithChildrenAttributeValues(c *gin.Context) {
 
 	for _, space := range spaces {
 		attributeID := entry.NewAttributeID(pluginID, inQuery.AttributeName)
-		attributeValue, ok := space.GetSpaceAttributeValue(attributeID)
-		if !ok {
-			err := errors.Errorf("Node: apiGetSpaceWithChildrenAttributeValues: space attribute value not found: %s", attributeID)
-			api.AbortRequest(c, http.StatusNotFound, "attribute_not_found", err, n.log)
-			return
-		}
+		attributeValue, _ := space.GetSpaceAttributeValue(attributeID)
 
-		spaceAttributes[space.GetID()] = attributeValue
+		if attributeValue != nil {
+			spaceAttributes[space.GetID()] = attributeValue
+		}
 	}
```

Comment:
controversial decision
in general you can't rely on the value returned with false ok flag
yes, right now if `ok == false`, then value is nil, but it might be some more cases when some "broken" value will be returned, for exmaple semi initialized
for me this code is a bit robuster
```
value, ok := space.GetSpaceAttributeValue()
if !ok {
    continue
}
```